workflows/post_process_run/scream_post/format_output.py

```diff
@@ -0,0 +1,186 @@
+        "omega": "upward_air_velocity",
+        "surf_sens_flux": "sensible_heat_flux",
+        "surf_evap": "latent_heat_flux",
+        "SW_clrsky_flux_up@bot": "clear_sky_upward_shortwave_flux_at_surface",
+        "SW_clrsky_flux_dn@bot": "clear_sky_downward_shortwave_flux_at_surface",
+        "LW_clrsky_flux_dn@bot": "clear_sky_downward_longwave_flux_at_surface",
+        "LW_clrsky_flux_up@bot": "clear_sky_upward_longwave_flux_at_surface",
+        "SW_clrsky_flux_up@tom": "clear_sky_upward_shortwave_flux_at_top_of_atmosphere",
+        "LW_clrsky_flux_up@tom": "clear_sky_upward_longwave_flux_at_top_of_atmosphere",
+        "VerticalLayerInterface": "vertical_thickness_of_atmospheric_layer",
+        # "pseudo_density": "pressure_thickness_of_atmospheric_layer",
+    }
+    ds = ds[output_variables].rename(rename_vars)
+    ds = ds.drop("horiz_winds")
+    ds["x_wind"] = u
+    ds["y_wind"] = v
+    ds["pressure_thickness_of_atmospheric_layer"] = make_delp(
+        ds.p_mid, ds.vertical_thickness_of_atmospheric_layer, ds.ps
+    )
+
+    ds[
+        "pressure_thickness_of_atmospheric_layer_tendency_due_to_nudging"
+    ] = make_placeholder_data(
+        ds.air_temperature,
+        "pressure_thickness_of_atmospheric_layer_tendency_due_to_nudging",
+        1e-3,
+    )
+    ds["air_temperature_tendency_due_to_nudging"] = make_placeholder_data(
+        ds.air_temperature, "air_temperature_tendency_due_to_nudging", 1e-3
+    )
+    ds["specific_humidity_tendency_due_to_nudging"] = make_placeholder_data(
+        ds.air_temperature, "specific_humidity_tendency_due_to_nudging", 1e-7
+    )
+    ds["x_wind_tendency_due_to_nudging"] = make_placeholder_data(
+        ds.air_temperature, "x_wind_tendency_due_to_nudging", 1e-3
+    )
+    ds["y_wind_tendency_due_to_nudging"] = make_placeholder_data(
+        ds.air_temperature, "y_wind_tendency_due_to_nudging", 1e-3
+    )
+    ds["tendency_of_air_temperature_due_to_scream_physics"] = make_placeholder_data(
+        ds.air_temperature, "tendency_of_air_temperature_due_to_scream_physics", 1e-5
+    )
+    ds["tendency_of_specific_humidity_due_to_scream_physics"] = make_placeholder_data(
+        ds.air_temperature, "tendency_of_specific_humidity_due_to_scream_physics", 1e-8
+    )
+    ds["tendency_of_eastward_wind_due_to_scream_physics"] = make_placeholder_data(
+        ds.air_temperature, "tendency_of_eastward_wind_due_to_scream_physics", 1e-5
+    )
+    ds["tendency_of_northward_wind_due_to_scream_physics"] = make_placeholder_data(
+        ds.air_temperature, "tendency_of_northward_wind_due_to_scream_physics", 1e-5
+    )
+
+    ds = ds.rename({"lev": "z", "ilev": "z_interface"})
+    ds = ds.assign_coords(
+        {
+            "z": np.arange(0, levs),
+            "z_interface": np.arange(0, levs + 1),
+            "ncol": np.arange(0, Ncols),
+            "tile": np.arange(0, 6),
+            "x": np.arange(0, x_dim),
+            "y": np.arange(0, y_dim),
+        }
+    )
```

Comment:
Is it necessary to assign these coordinates, or can we get away without it?

Reply:
We can get way without it, removed.